coast/config/config_parser.py

```diff
@@ -89,7 +89,16 @@ def _get_datafile_object(json_content: dict, data_file_type: str) -> Union[Datas
         dataset_var = dataset_json[ConfigKeys.VAR_MAP]
         dataset_dim = dataset_json[ConfigKeys.DIM_MAP]
 
+        try:
+            dataset_coord_vars = dataset_json[ConfigKeys.COO_VAR]
+            dataset_keep_all_vars = dataset_json[ConfigKeys.KEEP_ALL_VARS] \
+                if ConfigKeys.KEEP_ALL_VARS in dataset_json else "False"
```

Comment:
Do we need this check? dataset_json[ConfigKeys.KEEP_ALL_VARS] will throw a KeyError if the key doesn't exist, so could this be set to false in the except block?

Reply:
agreed and actioned